<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>First</class>
 <widget class="QDialog" name="First">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-10</y>
     <width>951</width>
     <height>591</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(243, 246, 249) ;</string>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>410</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#4f9dec, stop: 1 #4f9dec);
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>Mailling</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>0</y>
       <width>241</width>
       <height>601</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 40, 53) ;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#4f9dec, stop: 1 #4f9dec);
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>Exel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#4f9dec, stop: 1 #4f9dec);
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>Delete a fracture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>380</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#4f9dec, stop: 1 #4f9dec);
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>statistics</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#4f9dec, stop: 1 #4f9dec);
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>Add Fracture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>121</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#4f9dec, stop: 1 #4f9dec);
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>Modify a fracture</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>171</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(34, 182, 110);
background-color:  QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0#1e2835, stop: 1 #1e2835);
font: 75 8pt &quot;MS Shell Dlg 2&quot;;
 border-radius: 10;</string>
     </property>
     <property name="text">
      <string>       Responsible of cashier</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>701</width>
       <height>411</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QStackedWidget" name="stackedWidget_2">
      <property name="geometry">
       <rect>
        <x>-30</x>
        <y>0</y>
        <width>731</width>
        <height>421</height>
       </rect>
      </property>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>90</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Ref√©reance</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>210</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Montant</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_ref_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>90</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_rem_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>250</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>250</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Remise</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="line_date2_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>170</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="ajout_button_2">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>290</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Ajouter</string>
        </property>
       </widget>
       <widget class="QTableView" name="tableView222_4">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>20</y>
          <width>481</width>
          <height>371</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_id_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>130</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="line_mont_2">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>210</y>
          <width>113</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>170</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QStackedWidget" name="stackedWidget_3">
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>0</y>
          <width>731</width>
          <height>421</height>
         </rect>
        </property>
        <widget class="QWidget" name="page_5">
         <widget class="QLabel" name="label_17">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>90</y>
            <width>71</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Ref√©reance</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_18">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>210</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Montant</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_ref_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>90</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_rem_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>250</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLabel" name="label_19">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>250</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Remise</string>
          </property>
         </widget>
         <widget class="QDateEdit" name="line_date2_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>170</y>
            <width>111</width>
            <height>22</height>
           </rect>
          </property>
         </widget>
         <widget class="QPushButton" name="ajout_button_3">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>290</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>Ajouter</string>
          </property>
         </widget>
         <widget class="QTableView" name="tableView222_5">
          <property name="geometry">
           <rect>
            <x>270</x>
            <y>20</y>
            <width>481</width>
            <height>371</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_20">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>130</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Type</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_id_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>130</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLineEdit" name="line_mont_3">
          <property name="geometry">
           <rect>
            <x>130</x>
            <y>210</y>
            <width>113</width>
            <height>20</height>
           </rect>
          </property>
         </widget>
         <widget class="QLabel" name="label_21">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>170</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="page_6"/>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4"/>
     </widget>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>20</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="toolTip">
      <string/>
     </property>
     <property name="statusTip">
      <string/>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>831</width>
       <height>591</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:  rgb(243, 246, 249) ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>831</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255) ;
font: 75 18pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string> Dashboard</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>10</y>
       <width>41</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255) ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>111.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>81</width>
       <height>81</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(30, 40, 53) ;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>3.png</pixmap>
     </property>
    </widget>
    <zorder>label_2</zorder>
    <zorder>label_3</zorder>
    <zorder>frame</zorder>
    <zorder>pushButton_11</zorder>
    <zorder>label</zorder>
    <zorder>pushButton_6</zorder>
    <zorder>pushButton_3</zorder>
    <zorder>pushButton_12</zorder>
    <zorder>pushButton_2</zorder>
    <zorder>pushButton_4</zorder>
    <zorder>lineEdit_2</zorder>
    <zorder>lineEdit</zorder>
    <zorder>label_4</zorder>
    <zorder>label_5</zorder>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
